src/com/chrisfolger/needsmoredojo/core/amd/define/DefineResolver.java

```diff
@@ -250,4 +253,15 @@ public void visitJSCallExpression(JSCallExpression expression)
 
         return listOfDefinesOrRequiresToVisit;
     }
+
+    private Boolean isValidImportBlockName(String name, Project project) {
+        DojoSettings settings = ServiceManager.getService(project, DojoSettings.class);
+        String[] defineBlockNames = settings.getImportBlockNames().split(",");
+        for (String defineBlockName:defineBlockNames) {
+            if (name.equals(defineBlockName)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Should also line#47 be changed to `Collections.addAll(parameters, function.getParameterVariables());`?